modules/Quiz/resources/IVQModule.js

```diff
@@ -53,11 +53,28 @@
                     }
                     else {
                         $.quizParams = data[1];
+                        var banSeekTrigger = false;
+                        var dataForBanSeek = {
```

Comment:
@MaksymKorniiov  
To me it looks that there is a redundant variable here. Do we need banSeekTrigger if we are also saving the same value in dataForBanSeek?